src/styles/blocks/buttom.css

```diff
@@ -0,0 +1,36 @@
+.buttom-buy {
+  display: flex;
+  justify-content: center;
+
+  width: 166px;
+  height: 40px;
+  margin: 16px;
+
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  line-height: 40px;
+  text-transform: uppercase;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.buttom-buy__link {
+  display: block;
+  text-decoration: none;
+  color: #fff;
+}
+
+.buttom-buy:hover {
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
You can add border to the div, where your link is with property transparent for border-color. See checklist (item 8).